<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceInfoToolView</class>
 <widget class="QWidget" name="SourceInfoToolView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Functions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="functionArgumentNamesCheck">
     <property name="text">
      <string>Show argument names at call site</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="functionDefaultValuesCheck">
     <property name="text">
      <string>Show default argument values</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Structs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="structFieldSizeCheck">
     <property name="text">
      <string>Show field size and padding</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Variables</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoTypeCheck">
     <property name="text">
      <string>Show type of infered variables</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enum</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="enumValueCheck">
     <property name="text">
      <string>Show enum constant values</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
